ios/browser/ui/webui/brave_web_ui_controller_factory.mm

```diff
@@ -39,30 +42,31 @@
   return std::make_unique<T>(web_ui, url);
 }
 
+WebUIIOSFactoryFunction GetUntrustedWebUIIOSFactoryFunction(const GURL& url) {
+  const std::string_view url_host = url.host_piece();
```

Comment:
I wonder if we should have a `DCHECK_EQ(url.scheme_piece(), kChromeUntrustedScheme)`